src/utils/bytes.js

```diff
@@ -62,6 +76,18 @@ export function utf8Encode(string) {
 }
 
 /**
+ * Turn a string into UTF-8 bytes.
+ * @param {string} string
+ * @return {!Uint8Array}
+ */
+export function utf8EncodeSync(string) {
+  if (typeof TextEncoder !== 'undefined') {
```

Comment:
a4a decodes large string (body of ads creatives) using this, so the performance of TextEncoder on large strings is worth keeping.